native/cocos/audio/common/utils/include/primitives.h

```diff
@@ -14,13 +14,17 @@
  * limitations under the License.
  */
 
-#ifndef COCOS_AUDIO_PRIMITIVES_H
-#define COCOS_AUDIO_PRIMITIVES_H
+#pragma once
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
+    #include <sys/cdefs.h>
+#elif CC_PLATFORM == CC_PLATFORM_WINDOWS
+    #include <sys/types.h>
+    #include "audio/include/AudioMacros.h"
```

Comment:
Why need to include `AudioMacros.h` on Windows?

Reply:
To use __BEGIN_DECLS marco as well as android.